docs/installation.rst

```diff
@@ -1679,23 +1679,39 @@ for the creation of projectors in UPF can introduce severe errors and inaccuraci
 Installing Quantum ESPRESSO and pw2qmcpack
 ------------------------------------------
 
-For trial wavefunctions obtained in a plane-wave basis, we mainly
-support QE. Note that ABINIT and QBox were supported historically
-and could be reactivated.
+For trial wavefunctions obtained in a plane-wave basis, we mainly support Quantum ESPRESSO (QE). QBox support is also available, and
+support was ABINIT was available historically and could be reactivated.
 
-QE stores wavefunctions in a nonstandard internal
-"save" format. To convert these to a conventional HDF5 format file
-we have developed a converter---pw2qmcpack---which is an add-on to the
-QE distribution.
+We recommend using the latest version of Quantum ESPRESSO.
 
+To convert the QE wavefunctions to the HDF5 format used by QMCPACK file we have developed a converter -- pw2qmcpack -- which is an
+add-on to the QE distribution.
 
-Quantum ESPRESSO (<=6.8)
-~~~~~~~~~~~~~~~~~~~~~~~~
+Quantum ESPRESSO (>7.0)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+pw2qmcpack is configured via a plugin as part of the Quantum ESPRESSO installation. Simply specify
+``-DQE_ENABLE_PLUGINS=pw2qmcpack`` as part of the CMake configure step. Full QE CMake documentation can be found at
+https://gitlab.com/QEF/q-e/-/wikis/Developers/CMake-build-system . Excepting for a very large change to QE, the converter is
```

Comment:
I actually don't quite understand the meaning of "Excepting for a very large change to QE," I think we merely change anything in QE.

Reply:
The converter will work unless the QE developers do a major refactoring or changes of data structures etc., i.e. a large change.